scripts/npm-install.sh

```diff
@@ -8,6 +8,7 @@ npm_install()
 {
     echo Obtaining npm dependencies
     set +e
+    npm install -g yarn
```

Comment:
do you need the -g ? ie, can it work with just a local install ? otherwise, we can sudo install yarn with the proper ppa
See https://yarnpkg.com/lang/en/docs/install/#linux-tab